src/lib.rs

```diff
@@ -1617,6 +1617,11 @@ pub unsafe trait FromZeroes {
     }
 }
 
+/// Deprecated: prefer [`FromZeros`] instead.
+#[deprecated(since = "0.8.0", note = "`FromZeroes` was renamed to `FromZeros`")]
+#[doc(hidden)]
+pub use FromZeros as FromZeroes;
```

Comment:
This is going to use-rename both the trait *and* the proc macro (if it's in scope from the `derive`).